src/Executables/ReduceCceWorldtube/ReduceCceWorldtube.cpp

```diff
@@ -291,7 +291,9 @@ int main(int argc, char** argv) {
       "input_file", boost::program_options::value<std::string>()->required(),
       "name of old CCE data file")(
       "output_file", boost::program_options::value<std::string>()->required(),
-      "output filename")(
+      "output filename")("apply_spec_corrections",
```

Comment:
Here too, I'm still unsure if the different name (`apply_spec_corrections` vs `fix_spec_normalization` elsewhere) indicates a different action or is two names for the same thing